Sources/Model/Localized.yml

```diff
@@ -183,230 +219,278 @@ studySummary(count, total):
     de: Lerne (count) von (total) Karteikarten
     it(total == "1"): Studio (count) di (total) Flashcard
     it: Studio (count) delle (total) Flashcard
+    pt_BR(total == "1"): Estudando (count) de (total) Flashcard
+    pt_BR:  Estudando (count) de (total) Flashcards
 
 startStudyMode:
     en: Start Study Mode
     de: Starte den Lernmodus
     it: Avvia la modalità studio
+    pt_BR: Iniciar Modo de Estudo
 
 studySettings:
     en: Study Settings
     de: Einstellungen zum Lernmodus
     it: Impostazioni di studio
+    pt_BR: Configurações de Estudo
 
 continueStudying:
     en: Continue Studying
     de: Lernen fortsetzen
     it: Continua a studiare
+    pt_BR: Continuar Estudando
 
 terminateStudyMode:
     en: Terminate Study Mode
     de: Lernmodus beenden
     it: Termina la modalità studio
+    pt_BR: Terminar Modo de Estudo
 
 _continue:
     en: Continue
     de: Fortfahren
     it: Continua
+    pt_BR: Continuar
 
 makeIncorrect:
     en: Make Incorrect
     de: Als falsch markieren
     it: Rendi errato
+    pt_BR: Deixar Incorreto
 
 makeCorrect:
     en: Make Correct
     de: Als richtig markieren
     it: Rendi corretto
+    pt_BR: Deixar Correto
 
 answer:
     en: Answer
     de: Antwort
     it: Risposta
+    pt_BR: Resposta
 
 solution:
     en: Solution
     de: Lösung
     it: Soluzione
+    pt_BR: Solução
 
 correct:
     en: Correct!
     de: Korrekt!
     it: Corretto!
+    pt_BR: Correto\"
```

Comment:
Is this a typo and should be an exclamation mark, or is this actually correct?

Reply:
Sorry, I don't even know how that happened, the characters are not even close to each other.